examples/eestream/serve-collected/main.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/vivint/infectious"
 
 	"storj.io/storj/pkg/eestream"
+	"storj.io/storj/pkg/encryption"
```

Comment:
Can you place it into `pkg/stream/encryption`?

Reply:
What's the rationale? We have no `/pkg/stream` package yet. Also, the capabilities of this package is not just for stream encryption. For example, as one of the next steps the Path type will also depend on it for encrypting the path components.

Reply:
Hmm. Is the similarity in stream encryption and path encryption incidental or necessary? (I really don't know).

If it really is that they will completely share the package and have the same dependencies. Yes then `pkg/encryption` will be a better place.

When they don't share that much, it might make sense to have different packages to avoid the rise in internal complexity. I also realized that there is also meta info encryption that might be separate as well.